awx/main/migrations/0175_db_monitoring.py

```diff
@@ -0,0 +1,175 @@
+# Generated by Django 3.2.16 on 2023-02-07 14:39
```

Comment:
Does this need to be a migration? It seems like lot of this is for a local docker development? (so e.g. it could go to the init script also?)

And e.g. the firehose script is also just local dev or perf testing. It will be easier to get this in if we'd scope it as local/dev tools changes. And only think later what needs to be in the product.

Reply:
The migration has been removed. These changes should only impact development.